### PR TITLE
Updated ReadMe

### PR DESCRIPTION
**Pull request by sdhamani**

### CODE REVIEW

**Comment by vvaibhavdesai** [comment]
This looks good

**Comment by sdhamani** [comment]
Hello,
Live Website link: https://serene-ui.netlify.app/
Please feel free to go through it
Thank You.

**Comment by purvasheth** [comment]
image


Add some margin/padding to `Getting Started`

File: app.js [hunk 1]
```diff
@@ -53,9 +33,13 @@ btnTScrollUp.addEventListener("click", function () {
 });
 
 btnScrollInstallation.addEventListener("click", function () {
-  var menuitem = document.getElementById("menu");
-  menuitem.style.display = "none";
-  myFunctiontoggle();
+  console.log("Ins");
+  console.log(screenWidth);
```
Comment (purvasheth) [inline app.js]: always remove `console.logs` from final code

File: styles.css [hunk 2]
```diff
@@ -442,14 +430,6 @@ Alerts  */
...
-/* .dismiss-card:hover {
-  box-shadow: 0 0 0 4px hsla(0, 0%, 80%), 0 0 0 6px hsla(0, 0%, 75%);
-  /* transform: translateZ(1);
-  opacity: 0.5; */
-/* z-index: 2; */
-/* box-shadow: 5px 10px #888888;
-} */
-
 .card-image {
   width: 100%;
```
Comment (purvasheth) [inline styles.css:456]: I think `max-width:100%` kiya toh `height:auto` is not necessary. Try it out!

File: styles.css [hunk 3]
```diff
@@ -468,15 +448,13 @@ Alerts  */
   color: black;
   font-size: 80%;
   font-weight: 700;
-  /* padding: 5px 10px; */
 }
 
 .card-dismiss-btn:hover {
   padding: 10px 15px;
```
Comment (purvasheth) [inline styles.css:475]: Use `rem` even for padding. Makes them more consistent.

**Comment by sdhamani** [comment]
Hello Purva ,
As discussed, it is a browser-specific issue so we can ignore it.
image
Thanks for the review.

**Review by purvasheth — COMMENTED** [review]
Hi @sdhamani,

Accha banaya hai. 
Naam bhi mast hai -"SERENE". Library dekhke ekdum "serene" laga. :)
Thode minor code-level comments bhi hai. Dekh lena ek baar.
Thode aur suggestions 
 -  make `border:transparent` or `border: var(primary-color)` for primary buttons.
 - primary and secondary buttons seem same. Make the secondary button different.

File: app.js (hunk 1 above)
Comment (sdhamani) [reply app.js]: Removed them. 
I Will keep this in mind.
Thanks.

File: styles.css (hunk 2 above)
Comment (sdhamani) [reply styles.css:456]: Ya, I guess that is surplus. Removed it too.

**Review by Akshaay9 — COMMENTED** [review]
Hello
The component Library Looks good
One suggestion
1) add ` Cursor:pointer` to left side of Navigation Bar (on List items)
2) Add some padding on Top and botoom of alert bar

File: styles.css (hunk 3 above)
Comment (sdhamani) [reply styles.css:475]: Yes, will avoid using px as suggested by Kushagra too.

**Comment by sdhamani** [comment]
> Hello
> The component Library Looks good
> One suggestion
> add ` Cursor:pointer` to left side of Navigation Bar (on List items)

Thanks, Akshay for reviewing the PR.
I will add the cursor to the items in left Navigation Bar .

**Review by Akshaay9 — APPROVED** [review]
1) on `badge-on-icon-class`
add this code
`
.badge-on-icon {
    height: 1.4rem;
    border-radius: 50%;
    position: absolute;
    top: 0px;
    left: 3.5rem;
    background-color: var(--primary-color);
    color: var(--button-text-color);
    font-size: 0.8rem;
    display: flex;
    justify-content: center;
    align-items: center;
    padding: 0.4rem;
`
It will make it look bigger and better

**Review by Akshaay9 — COMMENTED** [review]
FInal review -)
on buttons add
`Outline:none`
it removes unnecessary border/outline.
Your Component library Looks Really good
Great Job..

**Comment by sdhamani** [comment]
> 1. on `badge-on-icon-class`
>    add this code
>    `.badge-on-icon { width: -39.7rem; height: 1.4rem; border-radius: 50%; position: absolute; top: 0px; left: 3.5rem; background-color: var(--primary-color); color: var(--button-text-color); font-size: 0.8rem; display: flex; justify-content: center; align-items: center; padding: 0.4rem;`
>    It will make it look bigger and better

Ya added these styles and now it really looks impressive.
Thanks a lot. :)

**Comment by sdhamani** [comment]
> FInal review -)
> on buttons add
> `Outline:none`
> it removes unnecessary border/outline.
> Your Component library Looks Really good
> Great Job..

Thanks a lot, Akshay  :).
As discussed , I added border: none to buttons and it worked

**Comment by ishanjirety** [comment]
image

Avoid using var. Instead, you should use const or let.

**Comment by sdhamani** [comment]
> image
> 
> Avoid using var. Instead, you should use const or let.

Hi Ishan,
I have changed the keywords to let and const.
Thanks for reviewing.

**Comment by ishanjirety** [comment]
The code looks really good, the library is working properly. 
I think it's good to go.

**Comment by sdhamani** [comment]
> The code looks really good, the library is working properly.
> I think it's good to go.

Thanks Captain :)

**Comment by AnkitKarnAK** [comment]
Hey Sagar
1. Your primary & secondary button looks same unless user hovers on it. They should be different on first look instead of just on hover.
2. Links in footer have very less left-right margin space. moreover, you gave space between `Git Hub` & it looks like four different links.
Annotation 2021-03-29 024529

**Comment by sdhamani** [comment]
> Hey Sagar
> 
> 1. Your primary & secondary button looks same unless user hovers on it. They should be different on first look instead of just on hover.
> 2. Links in footer have very less left-right margin space. moreover, you gave space between `Git Hub` & it looks like four different links.
>    Annotation 2021-03-29 024529

Hi Ankit,
Thanks for reviewing.
Changed bg color of my secondary button and added margin to the footer too.
let me know if any changes are needed.

**Pull request merged**